Sources/PackageDescription/Target.swift

```diff
@@ -230,25 +233,26 @@ public final class Target {
         }
     }
 
-    /// Create a library or executable target.
+    /// Creates a library or executable target.
     ///
-    /// A target can either contain Swift or C-family source files and you can't
-    /// mix Swift and C-family source files within a target. A target is
-    /// considered to be an executable target if there is a `main.swift`,
-    /// `main.m`, `main.c`, or `main.cpp` file in the target's directory. All
-    /// other targets are considered to be library targets.
+    /// A target can either contain Swift or C-family source files. You can't
+    /// mix Swift and C-family source files within a target. The Swift package manager
+    /// considers a target to be an executable target if there's a `main.swift`,
+    /// `main.m`, `main.c`, or `main.cpp` file in the target's directory. It considers
+    /// all other targets to be library targets.
     ///
     /// - Parameters:
     ///   - name: The name of the target.
     ///   - dependencies: The dependencies of the target. A dependency can be another target in the package or a product from a package dependency.
-    ///   - path: The custom path for the target. By default, a targets sources are expected to be located in the predefined search paths,
-    ///       such as `[PackageRoot]/Sources/[TargetName]`.
-    ///       Do not escape the package root; that is, values like `../Foo` or `/Foo` are invalid.
-    ///   - exclude: A list of paths to files or directories that should not be considered source files. This path is relative to the target's directory.
+    ///   - path: The custom path for the target. By default, the Swift package manager expects a target's sources to reside at predefined search paths,
+    ///       for example, `[PackageRoot]/Sources/[TargetName]`.
+    ///       Don't escape the package root; that is, values like `../Foo` or `/Foo` are invalid.
+    ///   - exclude: A list of paths to files or directories that the Swift package manager shouldn't consider to be source or resource files.
+    ///       This path is relative to the target's directory.
     ///       This parameter has precedence over the `sources` parameter.
-    ///   - sources: An explicit list of source files. In case of a directory, the Swift Package Manager searches for valid source files
-    ///       recursively.
-    ///   - publicHeadersPath: The path to the directory containing public headers of a C-family target.
```

Comment:
Not sure if we are actually restricting `publicHeadersPath` to just library targets.

Reply:
It would make sense. Should I add this info?